config/serverless.yml

```diff
@@ -8,6 +8,59 @@ xpack.fleet.internal.activeAgentsSoftLimit: 25000
 xpack.fleet.internal.onlyAllowAgentUpgradeToKnownVersions: true
 xpack.fleet.internal.retrySetupOnBoot: true
 
+## Fine-tune the feature privileges.
+xpack.features.overrides:
+  dashboard:
+    privileges:
+      ### Dashboard's `All` feature privilege should implicitly grant `All` access to Maps and Visualize features.
+      all.composedOf:
+        - feature: "maps"
+          privileges: [ "all" ]
+        - feature: "visualize"
+          privileges: [ "all" ]
+      ### Dashboard's `Read` feature privilege should implicitly grant `Read` access to Maps and Visualize features.
+      ### Additionally, it should implicitly grant privilege to create short URLs in Visualize app.
+      read.composedOf:
+        - feature: "maps"
+          privileges: [ "read" ]
+        - feature: "visualize"
+          privileges: [ "read" ]
+    ### All Dashboard sub-feature privileges should be hidden: reporting capabilities will be granted via dedicated
+    ### Reporting feature and short URL sub-feature privilege should be granted for both `All` and `Read`.
```

Comment:
I'm not sure I follow this comment. I don't think of Reporting as something that is backed by a "dedicated feature." Reporting is a service that allows applications to be reportable, which is why it is declared as sub-features of apps.